test/cmd/sdn.sh

```diff
@@ -4,29 +4,82 @@ trap os::test::junit::reconcile_output EXIT
 
 os::test::junit::declare_suite_start "cmd/sdn"
 
+# ClusterNetworks
 os::cmd::expect_success 'oc get clusternetworks'
 os::cmd::expect_success_and_text 'oc get clusternetwork default -o jsonpath="{.pluginName}"' 'redhat/openshift-ovs-multitenant'
 os::cmd::expect_failure_and_text 'oc patch clusternetwork default -p "{\"network\": \"1.0.0.0/8\"}"' 'Invalid value'
 os::cmd::expect_failure_and_text 'oc patch clusternetwork default -p "{\"hostsubnetlength\": 22}"' 'Invalid value'
 os::cmd::expect_failure_and_text 'oc patch clusternetwork default -p "{\"serviceNetwork\": \"1.0.0.0/8\"}"' 'Invalid value'
 
+# NetNamespaces
 orig_project="$(oc project -q)"
 
 os::cmd::expect_success 'oc get netnamespaces'
 os::cmd::expect_success_and_text 'oc get netnamespace default -o jsonpath="{.netid}"' '^0$'
-os::cmd::expect_failure 'oc get netnamespace sdn-test'
-os::cmd::expect_success 'oc new-project sdn-test'
-os::cmd::expect_success 'oc get namespace sdn-test'
-os::cmd::try_until_success 'oc get netnamespace sdn-test'
-os::cmd::expect_success_and_not_text 'oc get netnamespace sdn-test -o jsonpath="{.netid}"' '^0$'
-os::cmd::expect_success 'oc delete namespace sdn-test'
-os::cmd::try_until_failure 'oc get netnamespace sdn-test'
+
+os::cmd::expect_failure 'oc get netnamespace sdn-test-1'
+os::cmd::expect_success 'oc new-project sdn-test-1'
+os::cmd::expect_success 'oc get namespace sdn-test-1'
+os::cmd::expect_failure 'oc get netnamespace sdn-test-2'
+os::cmd::expect_success 'oc new-project sdn-test-2'
+os::cmd::expect_success 'oc get namespace sdn-test-2'
+os::cmd::expect_failure 'oc get netnamespace sdn-test-3'
+os::cmd::expect_success 'oc new-project sdn-test-3'
+os::cmd::expect_success 'oc get namespace sdn-test-3'
+
+os::cmd::try_until_success 'oc get netnamespace sdn-test-1'
+os::cmd::try_until_success 'oc get netnamespace sdn-test-2'
+os::cmd::try_until_success 'oc get netnamespace sdn-test-3'
+orig_vnid1="$(oc get netnamespace sdn-test-1 -o jsonpath='{.netid}')"
+orig_vnid2="$(oc get netnamespace sdn-test-2 -o jsonpath='{.netid}')"
+orig_vnid3="$(oc get netnamespace sdn-test-3 -o jsonpath='{.netid}')"
+os::cmd::expect_success "test '$orig_vnid1' != 0"
```

Comment:
I know you've done it this way since you do more tests with the same value below, but I really don't like:
 - not wrapping API interactions with `os::cmd` since that's where the logging is really useful
 - testing the result of a `test`